webapp/templates/jury/partials/clarification_form.html.twig

```diff
@@ -33,3 +33,30 @@
 
 </div>
 {{ form_end(form) }}
+
+<script>
+    $(function () {
+        var $body = $('body');
+
+        $body.on('submit', 'form[name=jury_clarification]', function () {
+            var clarid = {{ origclar | json_encode(constant('JSON_HEX_TAG')) | raw }}.clarid;
```

Comment:
This should also work, doesn't it?

```suggestion
            var clarid = {{ origclar.clarid }};
```

Reply:
I'll give it a try. I remember it didn’t work initially, which is why I wrote it this way.